client/src/app/pages/applications/analysis-wizard/analysis-wizard.tsx

```diff
@@ -163,10 +194,9 @@ export const AnalysisWizard: React.FC<IAnalysisWizard> = ({
   const methods = useForm<AnalysisWizardFormValues>({
     defaultValues: {
       artifact: null,
-      mode: "source-code-deps",
+      mode: determineMode(applications),
       formLabels: [],
       selectedTargets: [],
-      // defaults will be passed as initialFilterValues to the table hook
```

Comment:
this comment is needed to explain why `targetFilters` are left undefined (instead of empty array)

Reply:
I returned to the previous state